gui/src/main.rs

```diff
@@ -81,22 +81,17 @@ fn run() -> Result<(), ApplicationError> {
             event_handler,
         )
         .map_err(ApplicationError::RunVmm)?;
-    }
 
-    // Run VMM launcher.
-    let vmm = match run_launcher()? {
-        Some(v) => v,
-        None => return Ok(()),
+        todo!()
+    } else {
+        run_launcher()?
```

Comment:
`run_launcher` will return `Ok(None)` if the user exit the window instead of clicking on `Start` button. In this case we need to return immediately without creating the screen.

Reply:
We will return right in the next code